validation-test/compiler_crashers_fixed/28505-failed-call-arguments-did-not-match-up.swift

```diff
@@ -5,6 +5,6 @@
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
-// RUN: not --crash %target-swift-frontend %s -emit-ir
+// RUN: not %target-swift-frontend %s -emit-ir
```

Comment:
Yeah, makes sense. The mismatched argument lists are something like (unresolved_type) and (x: (unresolved_type)) so we're not going to get far anyway.